code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -219,9 +139,181 @@
 		return FALSE
 	return TRUE
 
-/obj/item/cardboard_cutout/setDir(newdir)
-	newdir = SOUTH
-	return ..()
-
 /obj/item/cardboard_cutout/adaptive //Purchased by Syndicate agents, these cutouts are indistinguishable from normal cutouts but aren't discolored when their appearance is changed
 	deceptive = TRUE
+
+/proc/generate_cardboard_cutouts()
+	var/list/cutouts = list()
+	for(var/datum/cardboard_cutout/cutout as anything in subtypesof(/datum/cardboard_cutout))
+		cutout = new cutout()
+		cutouts[cutout.name] = cutout
+	GLOB.cardboard_cutouts = cutouts
+
+/datum/cardboard_cutout
```

Comment:
Autodoc

Reply:
Please put this whole thing into it's own file (same dir)

Reply:
that's silly

Reply:
humor me

Reply:
humor me 😃 (just the datum part)

Reply:
a file with purely datums should not go in the items folder, this keeps everything contained and easy to see

Reply:
that just screams for a need to reorganize everything, but w/e